crates/ra_hir/src/resolve.rs

```diff
@@ -56,7 +56,7 @@ pub enum Resolution {
 }
 
 impl Resolver {
-    pub fn resolve_name(&self, name: &Name) -> PerNs<Resolution> {
+    fn resolve_name(&self, name: &Name) -> PerNs<Resolution> {
```

Comment:
Given that it's unclear if this looks only at the scopes or in the item_map as well, it seems to be a good idea to make this private.

Reply:
it should be looking at the item map as well (if there's a `ModuleScope` on the stack), I think...

Reply:
Hm, so the bug is that we don't have a module scope active? Will look into this, thanks!